spec/models/exam_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe Exam, type: :model do
```

Comment:
```suggestion
RSpec.describe Exam do
```

Não precisa por o type, o RSpec já infere o tipo do teste pela pasta que ele ta